Iceberg-Plugin-GitHub.package/IceGitHubAPI.class/instance/newRequestTo..st

```diff
@@ -2,10 +2,8 @@ private factory
 newRequestTo: aString
 	^ ZnClient new 
 		in: [ :this | 
-			isAnonymous ifFalse: [ 
-				this
-					username: self credentials username 
-					password: self credentials password ] ]; 
+			isAnonymous ifFalse: [
+				self credentials applyToRequest: this ] ]; 
```

Comment:
Something strange happened in a fresh image. "self credentials" was actually an instance of IceTipInteractiveCredentialsProvider. I am not sure how this happened. It would have DNUed on username as well.